### PR TITLE
Varonis get alerted events

### PR DESCRIPTION
**Pull request by vkorenkov-varonis**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py [hunk 1]
```diff
@@ -55,6 +56,103 @@
...
+    'Event.EventOnUser.IsDisabled',
+    'Event.EventOnUser.IsLockout',
+    'Event.EventOnUser.SamAccountName',
+    'Event.EventOnUser.AccountType.Name',
+    'Event.Destination.IP',
+    'Event.Destination.DeviceName'
+]
+
+EVENT_OUTPUT = [
+    'Varonis.Event.ID',
```
Comment (vdobrotskyi-allstar) [inline Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:122]: Do we have this field in documentation?

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py [hunk 2]
```diff
@@ -287,7 +439,73 @@ def get_search_result_path(search_response: List[Any]) -> str:
...
+    :return: A converted item or None
+    :rtype: ``Any``
+    """
+    if item:
+        try:
+            return converter(item)
+        except Exception as e:
+            if error:
+                raise error
+            raise e
```
Comment (vdobrotskyi-allstar) [inline Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:481]: Probably should be just "raise" in order to preserve callstack like in C#

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
 ''' CONSTANTS '''
 
+SEARCH_RESULT_RETRIES = 10
```
Comment (vdobrotskyi-allstar) [inline Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:19]: Do you know, is there any delay between each search or it just do immediate call? What do you think about adding backoff_factor to http client?

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py [hunk 4]
```diff
@@ -287,7 +439,73 @@ def get_search_result_path(search_response: List[Any]) -> str:
...
+    outputs: List[Any] = []
+    for row in rows:
+        output = dict()
+        for i in range(0, len(columns)):
+            output[columns[i]] = row[i]
+        outputs.append(output)
+    return outputs
+
+
+def try_convert(item, converter, error=None):
```
Comment (vdobrotskyi-allstar) [inline Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:463]: Pycharm is complaining that error is missing in docstring

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py (hunk 1 above)
Comment (vkorenkov-varonis) [reply Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:122]: no, but we need some key field I guess. What do you think?

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py (hunk 2 above)
Comment (vkorenkov-varonis) [reply Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:481]: fixed

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py (hunk 3 above)
Comment (vkorenkov-varonis) [reply Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:19]: Added

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py (hunk 4 above)
Comment (vkorenkov-varonis) [reply Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py:463]: fixed, thanks

File: Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py (hunk 1 above)
Comment (vkorenkov-varonis) [reply Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py]: added # TODO need to clarify spec

**Review by vdobrotskyi-allstar — APPROVED** [review]

**Pull request merged**